dbt_platform_helper/templates/svc/manifest-public.yml

```diff
@@ -1,3 +1,4 @@
+# {% extra_header %}
```

Comment:
Do we want this in this file and manifest-backend? We do manually edit those don't we? i.e. adding ip-filter or tweaking env vars etc.

Reply:
Yes, true, I think I went a bit too far with the find / replace. Will update